docs/08_cli.md

```diff
@@ -0,0 +1,202 @@
+# Command Line Interface
+
+The Cadence CLI is a command-line tool to perform various tasks on a Cadence server. It can perform 
+domain operations such as register, update, and describe as well as workflow operations like start 
+workflow, show workflow history, and signal workflow.
+
+## Using the CLI
+
+The Cadence CLI can be used directly from the Docker Hub image *ubercadence/cli* or by building the CLI tool
```

Comment:
Do you think it worth to make CLI available in open homebrew?

Reply:
Ya, that would be nice. Docker method would still be needed on Windows but at least Mac users would have a better experience.